src/APL/vercelAPL.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable class-methods-use-this */
+import debugPkg from "debug";
+import fetch from "node-fetch";
+
+import { APL, AuthData } from "./apl";
+
+const debug = debugPkg.debug("VercelAPL");
+
+export const TOKEN_VARIABLE_NAME = "SALEOR_AUTH_TOKEN";
+export const DOMAIN_VARIABLE_NAME = "SALEOR_DOMAIN";
+export const SALEOR_REGISTER_APP_URL = "SALEOR_REGISTER_APP_URL";
+export const SALEOR_DEPLOYMENT_TOKEN = "SALEOR_DEPLOYMENT_TOKEN";
+
+const envAuthData = (): AuthData | undefined => {
```

Comment:
Can we name functions with verbs so they are explicit? In JS `envAuthData` can be a function but also some static object, with `getEnvAuthData` (or better `getEnvAuth`) its clear that this is a function